lib/block-supports/index.php

```diff
@@ -71,8 +77,8 @@ function gutenberg_apply_block_supports( $block_content, $block ) {
 		return $block_content;
 	}
 
-	$xpath      = new DOMXPath( $dom );
-	$block_root = $xpath->query( '/html/body/*' )[0];
+	$body_element = $dom->getElementByID( $body_id );
```

Comment:
I think it'd be sufficient to 

```suggestion
	$body_element = $dom->getElementsByTagName( 'body' )[0];
```

(which means requiring that no other `<body />` be prepended, which I think is a fair assumption).

Also fine if you wanna play it even safer though :+1:

Reply:
I don't like going through the NodeList to get the first element, but I don't feel strongly.

I've implemented an alternative in b252259e2c52ecff711f812f48d026db700f85b0 that traverses simple, known path to the body.